src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/AutomatedPanningMachine.java

```diff
@@ -70,6 +72,14 @@ public void onInteract(Player p, Block b) {
             }
```

Comment:
```java
ItemUtils.consumeItem(input, false);
```


I can't comment on it but this would definitely consume the input item before the event is even being called.